lib/test/renaming/renaming_service_test.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:surf_flutter_starter/src/config/config_parameter.dart';
+import 'package:surf_flutter_starter/src/services/directory_service.dart';
+import 'package:surf_flutter_starter/src/services/renaming_service.dart';
+import 'package:surf_flutter_starter/src/utils/replace_path_separators_x.dart';
+import 'package:surf_flutter_starter/test/renaming/res/android_gradle_res.dart';
+import 'package:surf_flutter_starter/test/renaming/res/common_file.dart';
+import 'package:surf_flutter_starter/test/renaming/res/debug_android_manifest.dart';
+import 'package:surf_flutter_starter/test/renaming/res/main_activity.dart';
+import 'package:surf_flutter_starter/test/renaming/res/main_android_manifest.dart';
+import 'package:surf_flutter_starter/test/renaming/res/plist.dart';
+import 'package:surf_flutter_starter/test/renaming/res/profile_android_manifest.dart';
+import 'package:surf_flutter_starter/test/renaming/res/test_res.dart';
+import 'package:test/test.dart';
+import 'package:mocktail/mocktail.dart';
+
+/// Mock for [IODirectoryService]
+class DirectoryServiceMock extends Mock implements DirectoryService {}
+
+void main() {
+  late final RenamingService renamingService;
+  final DirectoryService directoryService = DirectoryServiceMock();
+
+  final mainManifestPath =
+      '$testExePath\\$projectName\\${ProjectPath.androidMainManifestPath}'.replacePathSeparators();
+  final debugManifestPath =
+      '$testExePath\\$projectName\\${ProjectPath.androidDebugManifestPath}'.replacePathSeparators();
+  final profileManifestPath =
+      '$testExePath\\$projectName\\${ProjectPath.androidProfileManifestPath}'
+          .replacePathSeparators();
+  final gradlePath =
+      '$testExePath\\$projectName\\${ProjectPath.androidAppGradlePath}'.replacePathSeparators();
+  final kotlinPath =
+      '$testExePath\\$projectName\\${ProjectPath.oldKotlinPath}'.replacePathSeparators();
+  final activityPath =
+      '$testExePath\\$projectName\\android\\app\\src\\main\\kotlin\\com\\test\\app\\MainActivity.kt'
+          .replacePathSeparators();
+  final commonPath =
+      '$testExePath\\$projectName\\${ProjectPath.iosCommonPath}'.replacePathSeparators();
+  final plistPath =
+      '$testExePath\\$projectName\\${ProjectPath.iosInfoPlistPath}'.replacePathSeparators();
+
+  setUpAll(() {
```

Comment:
The setup code is verbose: The setup code in the setUpAll method is quite lengthy and repetitive, with many similar lines of code. This could make it difficult to maintain the tests in the future if changes are made to the implementation. Consider refactoring the setup code to make it more concise and easier to maintain.

Reply:
✅ tried to simplify code, extracted reusable methods